sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/Uri.java

```diff
@@ -230,11 +236,14 @@ public int getPriority() {
     }
 
     static class HealthStatusAndDiagnosticStringTuple {
-        private final String diagnsoticString;
+        private String diagnosticString;
         private final HealthStatus status;
-
         public HealthStatusAndDiagnosticStringTuple(URI uri, HealthStatus status) {
```

Comment:
it can call HealthStatusAndDiagnosticStringTuple(uri, status, false)

Reply:
changed